.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "daml"]
+	path = daml
+	url = git@github.com:digital-asset/daml.git
```

Comment:
for the public read-only repo I suggest we use https here, so we don't require an ssh key for the checkout.